viral/cache_2p_sessions.py

```
@@ -559,6 +533,35 @@ def get_tiff_metadata(
+    stack_length = tiff.shape()[0]
+    tiff_timestamps = [
+        float(
+            re.search(
+                r"frameTimestamps_sec\s*=\s*(-?\d+\.\d+)",
+                tiff.description(idx),
+            )[1]
+        )
+        for idx in range(tiff.shape()[0])
+    ]
+    if any([t is None for t in tiff_timestamps]):
+        raise ValueError("Could not extract all timestamps from tiff description")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)